packages/internal/src/node/createBundles.js

```diff
@@ -29,7 +30,13 @@ export const createBundlesFromAbsolute = async sourceBundles => {
 
   for (const args of cacheToArgs.values()) {
     console.log(BUNDLE_SOURCE_PROGRAM, ...args);
-    const { status } = spawnSync(prog, args, { stdio: 'inherit' });
+    const env = /** @type {NodeJS.ProcessEnv} */ (
+      /** @type {unknown} */ ({
+        __proto__: process.env,
```

Comment:
Why not
```suggestion
        ...process.env,
```
?

Reply:
Still hopeful someday I can bring on my print spawn helper that includes  printing explicit/overridden environment variables besides the command and arguments executed.

Reply:
> my print spawn helper

?

Reply:
https://github.com/Agoric/testnet-load-generator/blob/3952279dfea0c3db31aca5b2bd024b417f4050b0/runner/lib/helpers/child-process.js#L165-L199

Reply:
That would omit inherited env properties. Is that the point? Why do you want that?

Reply:
for printing yes, because they're likely not relevant.

Reply:
Ok. Resolving as is.